adapters/bigoad/bigoad.go

```diff
@@ -0,0 +1,156 @@
+package bigoad
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"text/template"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/macros"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint *template.Template
+}
+
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
+	template, err := template.New("endpointTemplate").Parse(config.Endpoint)
+	if err != nil {
+		return nil, fmt.Errorf("unable to parse endpoint url template: %v", err)
+	}
+
+	bidder := &adapter{
+		endpoint: template,
+	}
+
+	return bidder, nil
+}
+
+func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	bigoadExt, err := getImpExt(&request.Imp[0])
+	if err != nil {
+		return nil, []error{err}
+	}
+
```

Comment:
Based on line 35 - `bigoadExt, err := getImpExt(&request.Imp[0])` looks like `Bigoad` expects all imps to use same values for `Imp.Ext`

@BIGOAds please confirm above behaviour. If  all imps should have same values for `Imp.Ext` then create/update bidder docs PR to mention this expected behaviour

Reply:
How to mention this in bidder docs PR

Reply:
Create doc PR in https://github.com/prebid/prebid.github.io repo and link doc PR in this PR's description.

Reply:
https://github.com/prebid/prebid.github.io/pull/5349
This is my doc PR